src/LibraryWindow.vala

```diff
@@ -1010,6 +1024,18 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
         }
     }
 
+    private void action_play () {
+        play_media ();
```

Comment:
Can't we just give this method when defining the action?

Reply:
It will complains because these methods take an argument. That's something I think I'd like to address in a later commit is removing this "inhibit_notifications" argument since we should probably just be deciding that based on the focus state of the window. I need to look into what else calls this method though and figure out what cases they're inhibiting notifications under and if there's something I'm missing